proposals/revised-governance.md

```diff
@@ -0,0 +1,227 @@
+*wants* to participate in the administrative side of things to do so.
+
+### How does this effect the CTC?
+
+It doesn't really. The CTC processes are generally not impacted by this with
+exception only to how new working groups are chartered and how issues escalated
+to the TSC level are resolved. The proposal does not change how the CTC works
+at all.
+
+### Why would working groups nominate someone to the Admin team?
+
+This would be to ensure proper representation among the working groups when key
+decisions are made with regards to budget allocation and administrative actions
+such as moderation enforcement. It's entirely up to the working group to
+decide if this is something they need or want.
+
+### Why is the TSC Director the chair of the Admin team?
+
+This is primarily for logistical reasons. The Admin team will be responsible for
+dealing with budget allocations and administrative tasks that involve
+interfacing with the Node.js Foundation. The TSC Director will be the primary
+touch point for such activities. This also echoes the situation in the
+Community Committee where the individual member representatives on the board
+share the Community Committee chair responsibilities.
+
+### It will take longer to get things done!
+
+It shouldn't take any longer than it does today.
+
+### This just means we’ll have more Working Groups!
+
+Working groups will carry with them a certain amount of required process.
+Chartering a new working group would also require consensus of all the
+other working groups, which means that there would be a fairly high bar to
+adding a new working group. This would mean that the growth of the number of
+voting seats would be much slower and manageable than the current potentially
+unbounded growth.
+
+### One monthly TSC meeting in which anyone can participate sounds like chaos!
+
+It will require better logistical planning, yes, but other groups (such as
+TC-39 for instance) pull off such meetings with very little difficulty. It
+will be the responsibility of the TSC Chair to pull together the agenda, and
+to ensure that there is adequate time for each working group to provide an
+update on its recent activity and priorities, as well as time to discuss any
+open `tsc-agenda` items.
+
+### What would happen to the existing working groups?
+
+Existing working groups would essentially be rechartered under this new process
+with a few notable changes. Specifically:
+
+* New working groups would not be chartered solely by the CTC.
+* Working groups would be encouraged to nominate one of their members to the
+  Admin team. (preferably someone actually willing to do administrative stuff)
+* Working groups would be expected to have a bit more process around documenting
+  who their members are.
+* Several of the existing working groups *could* be restructured *solely for
+  the purpose of voting*. This restructuring *does not change* how the teams
+  actually get things done or how people participate in them:
```

Comment:
If the members of the existing WG's continued to operate as they do today, with teams for each of the groups that are being merged, then additional meetings that span all of the sub-teams would likely be required for anything related to a vote.  I don't see everybody from the build team wanting to be involved in all of the Website team discussions and vice versa.